lib/include/environment/freertos/rpmsg_env_specific.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 NXP
+ * All rights reserved.
+ *
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**************************************************************************
+ * FILE NAME
+ *
+ *       rpmsg_env_specific.h
+ *
+ * DESCRIPTION
+ *
+ *       This file contains FreeRTOS specific constructions.
+ *
+ **************************************************************************/
+#ifndef RPMSG_ENV_SPECIFIC_H_
+#define RPMSG_ENV_SPECIFIC_H_
+
+#if defined(RL_USE_STATIC_API) && (RL_USE_STATIC_API == 1)
```

Comment:
Also missing header file for definition of RL_USE_STATIC_API

Reply:
Agree, could you please add #include "rpmsg_default_config.h" into all rpmsg_env_specific.h files?

Reply:
I agree